cosmo/monitors/nuv_dark_monitor.py

```diff
@@ -0,0 +1,77 @@
+import numpy as np
+import plotly.graph_objs as go
+import plotly.express as px
+import datetime
+import pandas as pd
+
+from peewee import Model
+from monitorframe.monitor import BaseMonitor
+from astropy.time import Time
+from typing import List, Union
+
+from .data_models import NUVDarkDataModel
+from ..monitor_helpers import fit_line, convert_day_of_year, create_visibility, v2v3
+from .. import SETTINGS
+
+COS_MONITORING = SETTINGS['output']
+
+
+class NUVDarkMonitor(BaseMonitor):
+    data_model = NUVDarkDataModel
+    # TODO: update docs
+    # docs = "https://spacetelescope.github.io/cosmo/monitors.html#<darks>"
+    output = COS_MONITORING
+
```

Comment:
```suggestion
# Plot stuff
x = 'TIME'
y= 'DARK_RATE'
plot_type = 'scatter'
```

Reply:
^^ If you want to use the built-in basic plotting

Reply:
https://spacetelescope.github.io/monitor-framework/creating_monitors.html#defining-a-new-monitor